docs/getting-started.md

```diff
@@ -257,23 +206,30 @@ configuration file has the following fields:
 Start the containerd snapshotter
 
 ```bash
-$ ./naive_snapshotter -address /var/run/firecracker-containerd/naive-snapshotter.sock -path /tmp/fc-snapshot
+$ ./naive_snapshotter \
+  -address /var/run/firecracker-containerd/naive-snapshotter.sock \
+  -path /tmp/fc-snapshot
 ```
 
 In another terminal, start containerd
 
 ```bash
-sudo PATH=$PATH /usr/local/bin/containerd
+$ sudo PATH=$PATH /usr/local/bin/firecracker-containerd \
```

Comment:
Why do we need `PATH=$PATH` here?

Reply:
`/usr/local/bin` isn't necessarily on `$PATH` when running as root.

Reply:
firecracker-containerd needs to be able to locate the other binaries that we've installed in `/usr/local/bin`, including `containerd-shim-aws-firecracker` and `firecracker`